github/enterprise_audit_log.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// GetAuditLog gets the audit-log entries for an organization.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/orgs#get-the-audit-log-for-an-organization
```

Comment:
I think this URL should be different for the Enterprise version.

Reply:
Fixed, I had that as a placeholder till I found the real link.